src/components/pages/Navbar/NavbarFeatures/SignupButton.js

```diff
@@ -1,9 +1,19 @@
 import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function SignupButton() {
+  const { loginWithRedirect } = useAuth0();
+
   return (
-    <NavBtnLink to={`/signup`} key="signup">
+    <NavBtnLink
+      className="btn btn-primary btn-block"
+      onClick={() =>
```

Comment:
Nice job importing auth0 and getting the functionality to redirect and work correctly.